selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/WebDriverWaitFactory.java

```diff
@@ -43,6 +44,14 @@ public WebDriverWait get(int timeoutInSec) {
     return new WebDriverWait(seleniumWebDriver, timeoutInSec).withMessage(messageSupplier);
   }
 
+  public FluentWait<WebDriver> get(
+      int timeoutInSec, Class<? extends Throwable> ignoredExceptionType) {
+    return new FluentWait<WebDriver>(seleniumWebDriver)
+        .withTimeout(timeoutInSec, TimeUnit.SECONDS)
+        .pollingEvery(200, TimeUnit.MILLISECONDS)
```

Comment:
`.pollingEvery(200, TimeUnit.MILLISECONDS)` looks redundant so as default delay between the attempts is preset to 500 millisecs inside FluentWait itself.